packages/mui-material/src/InputBase/InputBase.js

```diff
@@ -390,13 +387,6 @@ const InputBase = React.forwardRef(function InputBase(inProps, ref) {
   }, [value, checkDirty, isControlled]);
 
   const handleFocus = (event) => {
-    // Fix a bug with IE11 where the focus/blur events are triggered
```

Comment:
Introduced in https://github.com/mui/material-ui/pull/12076/files#diff-a5f495d99c59823944e8c99682e25d509e79d427ab0442c71589da33e6f350f6R225

Tested it in Chrome, Firefox, and Safari. Screenshot in Chrome:


https://github.com/mui/material-ui/assets/16889233/d3bb06e8-ee1f-48fa-aa40-c0311db7daf8